terraform/environments/core-logging/monitoring.tf

```diff
@@ -2,3 +2,44 @@ module "core_monitoring" {
   source                     = "../../modules/core-monitoring"
   pagerduty_integration_keys = local.pagerduty_integration_keys
 }
+
+provider "aws" {
+  alias  = "eu-west-2"
+  region = "eu-west-2"
+}
+
+resource "aws_cloudwatch_log_group" "aws_route53_logs_com" {
+  provider = aws.eu-west-2
+
+  name              = "aws_route53_zone.logs_com.name"
+  retention_in_days = 365
+}
+
+data "aws_iam_policy_document" "route53-query-logging-policy" {
+  statement {
+    actions = [
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = ["arn:aws:logs:*:*:log-group:/aws/route53/*"]
+
+    principals {
+      identifiers = ["route53.amazonaws.com"]
+      type        = "Service"
+    }
+  }
+}
+
+resource "aws_cloudwatch_log_resource_policy" "route53-query-logging-policy" {
+  provider = aws.us-east-1
+
+  policy_document = data.aws_iam_policy_document.route53-query-logging-policy.json
+  policy_name     = "route53-query-logging-policy"
+}
+
+# Example Route53 zone with query logging
+
+resource "aws_route53_zone" "logs_com" {
```

Comment:
There's no need to create a new route53 zone is there?

Reply:
Forgot to remove this code